doc/sphinx/source/quickstart/installation.rst

```diff
@@ -187,7 +187,7 @@ container is advised.
 
 The ``esmvaltool-python`` diagnostics can be installed as follows:
 
-First, ensure mamba is pre-installed (see `Mamba installation`_ for more details).
+First, ensure mamba is pre-installed (see `Mamba installation <https://docs.esmvaltool.org/en/latest/quickstart/installation.html#mamba-installation>_` for more details).
```

Comment:
The link is not well [displayed](https://esmvaltool--2580.org.readthedocs.build/en/2580/quickstart/installation.html#installation-on-macosx). I think you could do:

```suggestion
First, ensure mamba is pre-installed (see `Mamba/Conda installation`_ for more details).
```